styles/Accordion.module.css

```diff
@@ -0,0 +1,59 @@
+/* Style the accordion section */
```

Comment:
I won't review CSS at all yet since its not done, but I like the BEM style for classnames is pretty good 👍

Reply:
yeah it looks really clean! I will adjust my CSS class names